packages/styleguide/src/components/Editor/config/elements/articlePreview/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react'
+import {
+  ArticlePreviewElement,
+  ElementConfigI,
+  ElementFormProps,
+} from '../../../custom-types'
+import { ArticlePreviewIcon } from '../../../../Icons'
+import { css } from 'glamor'
+import ColorPicker from '../../../Forms/ColorPicker'
+import RepoSearch from '../../../Forms/RepoSearch'
+import Checkbox from '../../../../Form/Checkbox'
+import { useSlate } from 'slate-react'
+import { Transforms } from 'slate'
+import Field from '../../../../Form/Field'
+import { isValidHttpUrl } from '../../../Core/helpers/text'
+
+const styles = {
+  container: css({
+    display: 'grid',
+    gridTemplateColumns: 'repeat(2, 1fr)',
+    gridColumnGap: 20,
+    width: '100%',
+    marginTop: 20,
+  }),
+}
+
+const Form: React.FC<ElementFormProps<ArticlePreviewElement>> = ({
+  element,
+  path,
+  onChange,
+}) => {
+  const editor = useSlate()
+  const [syncData, setSyncData] = useState<boolean>(true)
+  const href = element.href || ''
+  return (
+    <>
+      <div>
+        <Field
+          label='URL'
+          type='url'
+          error={
+            !!href && !isValidHttpUrl(href) && 'Geben sie eine gültige URL an'
+          }
+          value={href}
+          onChange={(_, value: string) => {
+            onChange({
+              href: value,
+            })
+          }}
+        />
+        <RepoSearch
+          onChange={({ value }) => {
+            const href = `https://github.com/${value.id}?autoSlug`
+            onChange({
+              href,
+            })
+            if (syncData) {
+              const meta = value.latestCommit.document.meta
+              if (meta.title) {
+                Transforms.insertText(editor, meta.title, {
+                  at: path.concat([1, 0]),
+                })
+              }
+              if (meta.shortTitle) {
+                Transforms.insertText(editor, meta.shortTitle, {
+                  at: path.concat([1, 1]),
+                })
+              }
```

Comment:
are you sure you want to use the `shortTitle` here? it would be more logical to be `description` – `title` & `description` are made to work together. `shortTitle` is made to work standalone.

Reply:
You are right, I changed it.